src/components/AddForm.jsx

```diff
@@ -1,31 +1,26 @@
 import { FcPlus } from 'react-icons/fc';
-import './AddForm.css'
+import './AddForm.css';
 
-const InputBook = () => {
-
-    return (
-      <>
-        <form className="form-container">
-          <input
-            type="text"
-            placeholder="Add Author"
-            className="input-text"
-          />
-           <input
-            type="text"
-            placeholder="Add Title"
-            className="input-text"
+function AddForm() {
+  return (
+    <>
+      <form className="form-container">
+        <input type="text" placeholder="Add Author" className="input-text" />
+        <br />
+        <br />
+        <input type="text" placeholder="Add Title" className="input-text" />
+        <br />
+        <br />
+        <button className="input-submit" type="button">
+          <FcPlus
+            style={{
+              fontSize: '20px',
+              marginTop: '2px',
+            }}
           />
```

Comment:
- [OPTIONA] It's always considered bad practice to ad your CSS from the js file, 🤔  Instead of inline styles, it's better to use a separate stylesheet or a CSS-in-JS library like styled-components or Emotion to handle your component's styling. This way, you can keep your styling separate from your component code, which makes it more reusable, maintainable, and scalable. 👍🏻